tests/monitoring/monitoring.go

```diff
@@ -141,19 +164,29 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", func() {
 
 	Context("Up metrics", func() {
 		BeforeEach(func() {
-			scales = make(map[string]*autoscalingv1.Scale, 1)
+			scales = make(map[string]*autoscalingv1.Scale, 2)
 			backupScale(virtOperatorDeploymentName)
+			backupScale(virtControllerDeploymentName)
 		})
 
 		AfterEach(func() {
 			revertScale(virtOperatorDeploymentName)
+			customizeComponentPatch(virtControllerDeploymentName, scales[virtControllerDeploymentName].Spec.Replicas)
```

Comment:
Is this what's being used in a regular testing environment?
Why not simply remove the `CustomizeComponents` from KubevirtCR?